builder/builder.go

```diff
@@ -470,30 +472,46 @@ func (b *Builder) BuildImage(format string, configFile string) error {
 	imagecmd := exec.Command("clr-installer", "-c", configFile, "--swupd-versionurl",
```

Comment:
Since we are not printing anything to stdout by default, lets have an info message here:
`log.Info(log.Mixer, "Building image. Please wait...")`